src/pages/en/editor-setup.md

```diff
@@ -0,0 +1,50 @@
+---
+layout: ~/layouts/MainLayout.astro
+setup: |
+  import Badge from '~/components/Badge.astro';
+title: Editor Setup
+description: Set up your editor to build with Astro.
+---
+You can build your Astro site with any code editor, installed locally or in the cloud!
+
+This page explains how to improve the experience of editing `.astro` files in supported editors.
+
+## VS Code Extension
+
+[VS Code](https://code.visualstudio.com) is a popular code editor that can be downloaded locally, and is also used by online platforms such as [GitHub Codespaces](https://github.com/features/codespaces), and [Gitpod](https://gitpod.io). 
```

Comment:
```suggestion
[VS Code](https://code.visualstudio.com) is a popular code editor that can be downloaded and operated locally as well as on online platforms such as [GitHub Codespaces](https://github.com/features/codespaces), and [Gitpod](https://gitpod.io). 
```

Reply:
OK, I'm gonna give you one!!! Enjoy this!

(But, I'm gonna change operated to used.)

Reply:
```suggestion
[VS Code](https://code.visualstudio.com) is a popular code editor that can be downloaded and used locally as well as on online platforms such as [GitHub Codespaces](https://github.com/features/codespaces), and [Gitpod](https://gitpod.io).
```